gustaf/spline/visualize.py

```diff
@@ -36,6 +36,9 @@ class SplineShowOption(options.ShowOption):
             (bool,),
         ),
         options.Option("vedo", "knots", "Show spline's knots.", (bool,)),
+        options.Option(
+            "vedo", "contours", "Show spline's contour - 2D edges.", (bool,)
```

Comment:
`knots` option does exactly this

Reply:
If this is still my implementation, the idea here was to only show the contour and no interior knotlines, making it easier to detect patch-boundaries

Reply:
ok. maybe we can name this outlines? with contours I'd expect plotting isolines of splinedata